packages/ckeditor5-page-break/tests/pagebreakediting.js

```diff
@@ -57,7 +57,7 @@ describe( 'PageBreakEditing', () => {
 		setModelData( model, '[<pageBreak></pageBreak>]' );
```

Comment:
The TC name is still `should have label as a UIElement` now it is a raw element.